e2e/tests/moveObjects.e2e.spec.js

```diff
@@ -27,12 +27,46 @@ This test suite is dedicated to tests which verify the basic operations surround
 const { test, expect } = require('@playwright/test');
 
 test.describe('Move item tests', () => {
-    test.fixme('Create a basic object and verify that it can be moved to another Folder', async ({ page }) => {
-        //Create and save Folder
-        //Create and save Domain Object
-        //Verify that the newly created domain object can be moved to Folder from Step 1.
-        //Verify that newly moved object appears in the correct point in Tree
-        //Verify that newly moved object appears correctly in Inspector panel
+    test('Create a basic object and verify that it can be moved to another Folder', async ({ page }) => {
+
+        await page.goto('http://localhost:8080/');

```

Comment:
We can just do
`await page.goto('/');`

Reply:
Fixed. I'll remember to change this next time.